src/providers/node.rs

```diff
@@ -280,10 +280,12 @@ impl NodeProvider {
         let lock_json = app.read_file("package-lock.json").unwrap_or_default();
         let yarn_lock = app.read_file("yarn.lock").unwrap_or_default();
         let pnpm_yaml = app.read_file("pnpm-lock.yaml").unwrap_or_default();
+        let bun_lock = app.read_file("bun.lockb").unwrap_or_default();
```

Comment:
The bun lockfile is in binary format so I don't think this will work

Reply:
Let me add another test to see.

Reply:
I would test it but it seems `bun` just gets stuck on installing canvas.

Reply:
So this works for checking if the package is in the lockfile, it just seems Bun doesn't support NAPI (so canvas cannot be used).

Reply:
Let's remove this check then if canvas can't be used

Reply:
I think we should keep it for future proofing. It doesn't work because of a bug in bun.